code/modules/mob/living/silicon/robot/modules/module_clerical.dm

```diff
@@ -68,7 +68,7 @@
 		var/datum/reagents/R = emag.create_reagents(50)
 		R.add_reagent(/decl/reagent/paralytics, 10)
 		R.add_reagent(/decl/reagent/sedatives, 15)
-		R.add_reagent(/decl/reagent/ethanol/iced_beer, 25)
+		R.add_reagent(/decl/reagent/ethanol/beer, 25)
```

Comment:
```suggestion
		R.add_reagent(/decl/reagent/drink/ice, 5)
		R.add_reagent(/decl/reagent/ethanol/beer, 20)
```
Numbers can be tweaked; before it would've been 4:1 beer to ice (as it is in the suggestion), I think, but now you just need *any* amount of ice, even <1u.